A manufacturing company claims that the mean breaking strength of the cables supplied s 1500 with a standard deviation of 100.A sample of size 9 cables tested is having mean breaking strength of 1600 with a standard deviation of 30. If the distribution of the mean breaking strength is normal, does this support the claim at 1% level of significance?
a)	Use z – statistic to validate the claim
b)	Use t – statistics to validate the
c)	Repeat the same at 5% level of significance
d)	Based on (a),(b) and (c) write your observations , comments and suggestions



## Answer

### Conditions for applying Z or t Hypothesis:
1. Sample needs to be random or normally distributed or the histogram of the sample distribution 
   should be somewhat normal with no outliers or skeweness.
2. Large sample size ( n > 30).
3. If the sample size is less than 30 but distribution is normal we can safely apply t-test and z-test.

### Observation 
1. It is given that the mean breaking strength is normal.
2. Although, the sample size is small ( n = 9 ) but it's distribution is normal 
   so we can apply z-test and t-test statistics.

## Steps for Hypothesis testing

1. Stating the null hypothesis and alternate hypothesis.
2. Collecting relevant data, checking that the data satisfy the conditions which allow us to use this test, 
   and summarizing the data using a test statistic.
3. Finding the p-value of the test, the probability of obtaining data as extreme as those
   collected (or even more extreme, in the direction of the alternative hypothesis), assuming that 
   the null hypothesis is true. In other words, how likely is it that the only reason for getting data 
   like those observed is sampling variability (and not because Ho is not true)?
4. Drawing conclusions, assessing the significance of the results based on the p-value, 
   and stating our conclusions in context. (Do we or don't we have evidence to reject and accept ?)



## Step 1:
$$ H_0 : \mu = 1500 $$
$$ H_1 : \mu \neq 1500 $$

## Step 2
$$ population \;mean \; \mu = 1500 $$
$$ population \;STD \; \sigma = 100 $$
$$ sample \;mean \; \bar x = 1600 $$
$$ sample \;STD  \; s = 30 $$
$$ sample \;size \; n = 9 $$

## Step 3 : 


In [24]:
## Calculate the z-score 
import math
import scipy.stats as st
mean_diff = 1600 - 1500
sample_variation = 100 / math.sqrt(9)
zscore = mean_diff/sample_variation
zpvalue = 2 * (1-st.norm.cdf(zscore))
print (round(zpvalue,4))

0.0027


In [26]:
## Compare with the critical t-value
#Degrees of freedom
df = 9 - 1
ttest = (mean_diff / 30) * math.sqrt(9-1)
print(ttest)
#p-value after comparison with the t 
p = 1 - st.t.cdf(ttest,df=df)
print("t = " + str(ttest))
print("p = " + str(2*p))

9.428090415820634
t = 9.428090415820634
p = 1.3152097325885848e-05


## For 1% significance level the z-score in 2 tailed test is 2.576
## For 5% significance level the z-score in 2 tailed test is 1.96

### since the z-score is 3 which is greater than both 2.576 and 1.96 , the Null hypothesis is rejected . Also , the p-value of 0.0027 is less than 0.01 and 0.05 which implies that the data present strong evidence to reject the NULL hypothesis 

## For 1% significance level the t-test  statistics in 2 tailed test  with 8 degree of freedom is 3.355
## For 5% significance level the t-test statistics in 2 tailed test  with 8 degree of freedomis 2.306

### since the t-test value  is 9.43 which is greater than both 3.355 and 2.306, the Null hypothesis is rejected . Also , the p-value of 0.00001315 is less than 0.01 and 0.05 which implies that the data present strong evidence to reject the NULL hypothesis 

# The  manufacturing company's claim that the mean breaking strength is 1500 can be rejected 